src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_SHORT_RENT = 20;
+  const DISCOUNT_LONG_RENT = 50;
+
+  const price = days * COST_PER_DAY;
+
+  if (days >= 7) {
```

Comment:
no magic numbers pls, move them all to consts
<img width="806" alt="transOnVacChlist" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/77551820/f1e32d53-07e1-44c7-bed5-d5cdaba6f62d">